norse/torch/module/lif.py

```diff
@@ -128,17 +138,29 @@ def __init__(
         self,
         input_size: int,
         hidden_size: int,
+        sparse: bool = False,
         p: LIFParameters = LIFParameters(),
         **kwargs
     ):
-        super().__init__(
-            activation=lif_step,
-            state_fallback=self.initial_state,
-            p=p,
-            input_size=input_size,
-            hidden_size=hidden_size,
-            **kwargs,
-        )
+        if sparse:
```

Comment:
At some point we should profile these kinds of checks. My intuition tells me it doesn't impact performance, but it'd be good to know. Fine for now, though.

Reply:
Yes, I agree.